process/README.md

```diff
@@ -43,23 +43,9 @@ See the [example configuration](https://github.com/DataDog/integrations-core/blo
 
 ### Validation
 
-[Run the Agent's `status` subcommand](https://docs.datadoghq.com/agent/faq/agent-commands/#agent-status-and-information) and look for `process` under the Checks section:
+[Run the Agent's `status` subcommand](https://docs.datadoghq.com/agent/faq/agent-commands/#agent-status-and-information) and look for `process` under the Checks section.
 
-```
-  Checks
-  ======
-    [...]
-
-    process
-    -------
-      - instance #0 [OK]
-      - instance #1 [OK]
-      - Collected 26 metrics, 0 events & 1 service check
-
-    [...]
-```
-
-Each instance configured in `process.yaml` should have one `instance #<num> [OK]` line in the output, regardless of how many search_strings it might be configured with.
+**Note**: Each instance configured in `process.yaml` should have one `instance #<num> [OK]` line in the output, regardless of how many search_strings it might be configured with.
```

Comment:
This note should also be removed or at least rephrased since it refers to `instance #<num> [OK]`, which syntax is specific to the agent 5 output.